use-current-runtime/test.json

```diff
@@ -7,5 +7,6 @@
   "type": "bash",
   "cleanup": true,
   "ignoredRIDs":[
+    "linux-s390x"
```

Comment:
This test makes sure that a restore/build with `--use-current-runtime` uses the portable RID id, `linux-s390x`, which isn't going to work because those assets are not available publicly.